lib/mix/tasks/migrate_db.ex

```diff
@@ -0,0 +1,80 @@
+defmodule Mix.Tasks.MigrateDb do
+  use Mix.Task
+
+  require Ex2ms
+
+  alias AeMdw.Db.Model
+  alias AeMdw.Db.Util
+  alias AeMdw.Log
+
+  @table Model.Migrations
+  @record_name Model.record(@table)
+  @version_len 14
+
+  @migrations_code_path "priv/migrations/*.ex"
+
+  @impl Mix.Task
+  def run(_) do
+    :net_kernel.start([:aeternity@localhost, :shortnames])
+    :ae_plugin_utils.start_aecore()
+    :lager.set_loglevel(:epoch_sync_lager_event, :lager_console_backend, :undefined, :error)
+    :lager.set_loglevel(:lager_console_backend, :error)
+    Log.info("================================================================================")
+    current_version = read_migration_version(@table)
+    Log.info("current migration version: #{current_version}")
+
+    applied_count =
+      @migrations_code_path
+      |> list_migrations_modules()
+      |> Enum.map(&maybe_apply_migration(&1, current_version))
+      |> Enum.count(&(&1))
+
+    # assure filesystem sync
+    if applied_count > 0, do: :mnesia.dump_log()
+
+    System.stop(0)
```

Comment:
This isn't really needed, it's the default

Reply:
The reset_db had it. Supposedly would help stopping the plugin but indeed without it console comes fast too.